acceptance_tests/features/steps/notification_letter.py

```diff
@@ -0,0 +1,54 @@
+from datetime import timedelta, datetime
+from time import sleep
+
+import paramiko
+from behave import given, when, then
+
+from acceptance_tests.features.environment import poll_database_for_iac
+from config import Config
+from controllers.collection_exercise_controller import create_and_execute_collection_exercise
+
+
+@given('a reporting unit has been enrolled in a survey')
+def reporting_unit_enrolled(context):
+    now = datetime.utcnow()
+    context.start = datetime.now()
+    dates = {
+        "mps": now + timedelta(seconds=5),
+        "go_live": now + timedelta(seconds=10),
+        "return_by": now + timedelta(days=10),
+        "exercise_end": now + timedelta(days=11),
+    }
+    # Create action rule
+    create_and_execute_collection_exercise('cb8accda-6118-4d3b-85a3-149e28960c54', '0718', 'Testing notification file',
+                                           dates, short_name='Bricks')
+
+
+@when('the survey goes live')
+def survey_is_live(context):
+    context.iac_code = poll_database_for_iac('cb8accda-6118-4d3b-85a3-149e28960c54', '0718')
+
+
+@then('the reporting unit will receive a letter')
+def letter_is_received(context):
+    content, time = get_latest_file()
+    while datetime.fromtimestamp(time) < context.start:
```

Comment:
Will this not hang if the file is not generated? Might be better to have it fail instead to avoid someone realising the build has been stuck for an hour if the file isn't created.